envFiles/.env.rootless.devcontainer

```
@@ -0,0 +1,136 @@
+API_COMMUNITY_SLACK_URL=https://slack.com
+API_COMMUNITY_WEBSITE_URL=https://docs.talawa.com
+API_COMMUNITY_X_URL=https://x.com
+API_COMMUNITY_YOUTUBE_URL=https://youtube.com
+API_DEBUGGER_HOST=0.0.0.0
+API_DEBUGGER_PORT=9229
+API_HOST=0.0.0.0
+API_IS_APPLY_DRIZZLE_MIGRATIONS=true
+API_IS_GRAPHIQL=true
+API_IS_PINO_PRETTY=true
+API_JWT_EXPIRES_IN=2592000000
+API_JWT_SECRET=b4896453be722d5ca94058a73f52b31c75980b485fa6d74d91f417a8059d8731
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The API_JWT_SECRET key should go before the AWS_ACCESS_KEY_ID key

(UnorderedKey)